registry/proxy/proxyregistry.go

```diff
@@ -193,8 +193,9 @@ func (r *remoteAuthChallenger) tryEstablishChallenges(ctx context.Context) error
 	r.Lock()
 	defer r.Unlock()
 
-	remoteURL := r.remoteURL + "/v2/"
-	challenges, err := r.cm.GetChallenges(remoteURL)
+	remoteURL := r.remoteURL
+	remoteURL.Path = "/v2"
```

Comment:
:+1: for trailing slash, but at least engine behavior currently enforces that `/v2/` is the top-level path